templates/client/list.html.twig

```diff
@@ -5,20 +5,10 @@
 
         <h1>Nos services</h1>
         <div class="services-list">
-            <ul>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-            </ul>
+            {#
+            {% for service in services %}
+                < class="bubble-service"><p>{{ service.name }}</p></>
+            {% endfor %} #}
```

Comment:
You shouldn't commit dead code

Reply:
Ok this is because this pull request anticipate the next one i have just send.